nautobot/docs/administration/celery-queues.md

```diff
@@ -0,0 +1,36 @@
+# Celery Worker Queues
+
+If you're planning to run multiple jobs, leverage job hooks or are finding that your jobs are taking too long to complete you may want to consider deploying multiple celery workers with different queues for different types of tasks.
+
+## How Celery Task Queues Work
+
+If you don't change any of the defaults when [deploying a celery worker](../installation/services.md#celery-worker) it will listen to the default queue named `celery`, use a [`--concurrency` value](https://docs.celeryq.dev/en/stable/reference/cli.html#cmdoption-celery-worker-c) set to the number of CPUs detected on the system and the [`worker_prefetch_multiplier` setting](https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-worker_prefetch_multiplier) defaults to 4. This means that a worker running on a 4 core system will run 4 tasks concurrently and reserve a maximum of 16 more tasks from the queue. If you have a mixture of long running and short running tasks with a single queue, you could find your long running tasks blocking the shorter tasks.
```

Comment:
This is a bit of a run-on sentence, and I'm not sure if this is supposed to be a list of default behaviors, or something else. If this is a list of default behaviors, then I think it would be better as a list

Reply:
This looks a little bit better to me. What do you think?

## How Celery Task Queues Work

The default celery behavior is:

- [`--queue celery`](https://docs.celeryq.dev/en/stable/reference/cli.html#cmdoption-celery-worker-Q)
- [`--concurrency`](https://docs.celeryq.dev/en/stable/reference/cli.html#cmdoption-celery-worker-c) set to the number of CPUs detected on the system
- [`worker_prefetch_multiplier=4`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-worker_prefetch_multiplier)
 
This means that a worker running on a 4 core system will run 4 tasks concurrently and reserve a maximum of 16 more tasks from the queue. If you have a mixture of long running and short running tasks with a single queue, you could find your long running tasks blocking the shorter tasks.

Reply:
That does look better, but I would change `from the queue` to `from the queue named celery` to be clear about it is only listening to that particular queue